src/calculateRentalCost.js

```diff
@@ -4,7 +4,18 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const carRentPerDay = 40;
+  const saleForWeekAndMore = 50;
+  const saleForShortTerm = 20;
+  let totalSum = carRentPerDay * days;
+
+  if (days >= 7) {
```

Comment:
2. [checklist] Don't use literals in expressions (HARDCODE). Instead use constants explaining this values
abount
and for the next line